server/config/routes.js

```diff
@@ -784,6 +784,8 @@ exports.configure = function configure(app) {
   // @todo - this should use the JSON renderer instead of it's own route!
   app.get('/finance/cashflow', financeReports.cashflow.report);
 
+  app.get('/status', stock.listStatus);
```

Comment:
These are only stock status's , right?  Could you make this route `/stock/status` to reflect that?